### PR TITLE
Fix main sanity check not running update_old_propers DUH!

### PR DESCRIPTION
**Pull request by fernandog**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/databases/main_db.py [hunk 1]
```diff
@@ -260,6 +262,7 @@ def fix_unaired_episodes(self):
                                    [common.UNAIRED, cur_unaired["episode_id"]])
 
     def fix_tvrage_show_statues(self):
+        # Remove?
         status_map = {
             'returning series': 'Continuing',
             'canceled/ended': 'Ended',
```
Comment (fernandog) [inline medusa/databases/main_db.py]: @p0psicles do we have something similar to the others indexers?

"tvmaze" has the 'to be determined' status. Its not 'ended' but its not 'continuing'
Keep as it is or make a list of default medusa show status so we map all?

File: medusa/databases/main_db.py [hunk 2]
```diff
@@ -260,6 +262,7 @@ def fix_unaired_episodes(self):
                                    [common.UNAIRED, cur_unaired["episode_id"]])
 
     def fix_tvrage_show_statues(self):
+        #  Remove?
```
Comment (p0psicles) [inline medusa/databases/main_db.py]: No this one can stay. Still need to check all the show status we currently have. As I didn't sanitize these. Then we just rename this one.

**Comment by p0psicles** [comment]
But how is this really a bug related to propers?

**Comment by fernandog** [comment]
Its not updating old propers with proper tags (before we added the proper tags code)

File: medusa/databases/main_db.py (hunk 2 above)
Comment (fernandog) [inline medusa/databases/main_db.py]: Rename then? Remove tvrage from name

**Comment by fernandog** [comment]
waiting your OK @p0psicles

**Pull request merged**